index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="/styles/style.css">
+    <link rel = "icon" href =  "/pict-mobile/bxs-radiation.svg" type = "image/x-icon">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="top-page">
+            <div class="ornament-top"><img src="/pict-mobile/image_geometry_1.svg" alt=""></div>
+            <div class="menu1"><img src="/pict-mobile/ic_menu.svg" alt=""></div>
+        </div>
+            <div class="top-text"> 
+                <h1 class="top-name">ELDON RATZLAFF</h1>
+                <p class="top-coment">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                  If you like what you see and have a project you need coded, don’t hesitate and contact me.
+                 </p>
+        
+                <div class="top-buton">
+                Start collaboration
+                </div>
+            </div> 
+            <div class="ornamen-first-botom">
+            </div>       
```

Comment:
Since this is the headline section, I think it will be a great idea to pull this out of the header tag and make it a standalone item wrapped inside of a `section` tag.

Like this...

```
<section>
  Headline content
</section>
```

Also, as required, I suggest aligning the contents in the headline using flex-box since they help when building a flexible layout like this one.